docs/product-guide/vsan/storage-tiers.md

```diff
@@ -0,0 +1,206 @@
+---
+status: new
+---
+
+# Storage Tiers in VergeOS vSAN
+
+## Overview
+
+VergeOS vSAN implements a sophisticated storage tiering system that balances performance, capacity, and cost. The system supports up to 6 tiers (0-5), each optimized for different workloads and data types. This tiered architecture allows organizations to efficiently manage their storage resources by placing data on the most appropriate tier based on performance requirements and access patterns.
+
+## Tier Specifications
+
+### Tier 0: Metadata Tier
```

Comment:
Should we put in the guidelines for T0 sizing here or a link to the sizing doc?

Reply:
updated after overview section